### PR TITLE
Added key replacement method and refactored file tree for testing

### PR DESCRIPTION
**Pull request by ricdezen**
# Premise
The pull request regards two topics:
- The addition of a regex replacement method in `key_handler.py`, not much to say here
- The refactoring of the file tree in something that (should) not break.

### Structure
The structure is as follows
```
root
├── download.py
├── upload.py
├── run_all_tests.py
├── config.json
├── README.md
├── ...
├── otri
│   ├── downloader
│   │   ├── timeseries_downloader.py
│   │   └── ...
│   ├── importer
│   │   └── ...
│   └── ...
├── test
│   └── ...
├── docs
│   └── ...
└── data
    └── ...
```
### Imports
The imports always assume the working directory to be outside of the `otri` directory, just as it would be when using the module from outside. So the scripts like download.py and such, refer to directories/files starting from the root and have imports that look like:
```python
import otri.downloader.timeseries_downloader
from otri.database import *
```
Tests use absolute imports too, just like above, assuming we will always start them from the root directory.

Actual source files use relative imports:
```python
# Another module from the same package/subpackage
import .timeseries_downloader
# A module from another subpackage in the parent package
from ..database import database_adapter
```

I have tried downloading, uploading and running the very few tests I made, and everything seems to work.
Please note that the VSCode `python.terminal.runFilesInDir` option disrupts the logic of the file tree because it changes the working directory, which I remind should be the root.

### CODE REVIEW

**Comment by CremaLuca** [comment]
> Please note that the VSCode `python.terminal.runFilesInDir` option disrupts the logic of the file tree

So does that option have to be checked or not?

**Comment by ricdezen** [comment]
Should NOT be checked

File: otri/utils/key_handler.py [hunk 1]
```diff
@@ -97,16 +103,30 @@ def rename_deep(data, aliases : dict):
...
+        aliases : dict
+            Dictionary containing the aliases for the keys. For each item the key must be
+            the regex to replace and the value what to replace it with.
+            Only string keys are modified.
+    Returns:
+        A copy of the dict or list with the renamed keys, with all nested dicts and lists
+        receiving the same treatment. It will return the original object (not a copy)
+        if no operation could be applied. See apply_deep(data, fun) for details.
+    '''
+    def replace_regex(string):
```
Comment (CremaLuca) [inline otri/utils/key_handler.py]: Nice and clean, really good work

File: run_all_tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import os
+
+os.system('cmd /c "python -m pytest"')
```
Comment (CremaLuca) [inline run_all_tests.py:3]: I'm interested: what does `cmd /c` do?

**Review by CremaLuca — APPROVED** [review]
Probably the perfect solution, easy and elegant, gj.

**Comment by CremaLuca** [comment]
You might wanna add tests for the key replace method

File: run_all_tests.py (hunk 2 above)
Comment (ricdezen) [reply run_all_tests.py:3]: `/c` is basically "exit after the process is done", not sure it would change much here

**Pull request merged**